_posts/2019-08-27-building-a-cli-for-firmware-projects.md

```diff
@@ -321,7 +321,7 @@ overridden by our environment, and now we can define this variable ourselves
 from our Invoke task!
 
 ```
-# ⁨nRF5_SDK_15.2.0/components⁩/toolchain⁩/gcc⁩/Makefile.common
+# nRF5_SDK_15.2.0/components/toolchain/gcc/Makefile.common
```

Comment:
Wow how did you discover this 😮 !?

Reply:
😅 some truly horrific scripting when debugging lunr.js index

Reply:
😅 😬